modules/serverless-functions-quarkus-return-value-types.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-quarkus-functions.adoc
+
+[id="serverless-functions-quarkus-return-value-types_{context}"]
+= Permitted types
+
+The input and output types of a function can be any of the following:
+
+* `void`
+* `String`
+* `byte[]`
+* Primitive types and their wrappers (for example, `int` and `Integer`).
+* A JavaBean, if its attributes are of types listed here.
+* A map, list, or array of the types in this list.
+* The special `CloudEvents<T>` type, where the `<T>` type parameter is of a type in this list.
```

Comment:
Hmm.. I think here is a little confusing.. 
"... where `<T>` type parameter is of a type in this list."

what about ... 

"... where the `<T>` type parameter is one of the permitted types above."
, wdyt?

Reply:
It's actually in the IBM style guide that we shouldn't use _above_.

> above adjective, adverb, preposition
> 1. Do not use to refer to a subsequent software version or fix level. Use "later". 2. Do not use to indicate a
> relative location in a document, as in "the above restrictions" or "the steps above". Use "previous" or
> "preceding". Examples: "The previous restrictions apply to ..."; "The preceding steps..."

I think referring to the list makes the most sense otherwise, since it's worded that way in the previous bullet, but I'll wait to see if there's any additional guidance or feedback on this from the OpenShift docs peer review if that's OK 🙂

Reply:
Sure, perfect